README.md

```diff
@@ -53,11 +62,11 @@ In addition to the above, the plugin reports the following custom metrics, which
 
 
 # Quality Rules and Profiles
-This plugin integrates 69 quality checks from [Scalastyle](http://www.scalastyle.org/rules-1.0.0.html) and 117 from [Scapegoat](https://github.com/sksamuel/scapegoat/tree/v1.3.7). 40 of the Scalastyle rules are defined without parameters and the remaining 29 are rule templates, which allow you to set up custom rules by specifying the parameters yourself. Most of those are are already activated for you with default values recommended by Scalastyle. Scapegoat rules don't contain any templates.
+This plugin integrates 69 quality checks from [Scalastyle](http://www.scalastyle.org/rules-1.0.0.html) and 117 from [Scapegoat](https://github.com/sksamuel/scapegoat/tree/v1.3.8). 40 of the Scalastyle rules are defined without parameters, and the remaining 29 are rule templates, which allow you to set up custom rules by specifying the parameters yourself. Most of those are already activated for you with default values recommended by Scalastyle. Scapegoat rules don't contain any templates.
 
-Sonar-scala creates two rule repositories and four built-in quality profiles: `Scalastyle`, `Scapegoat` and a combination of those two `Scalastyle+Scapegoat` and `Recommended by sonar-scala`. The rules in the quality profiles are almost all activated, except for those which are broken in the upstream projects and had to be blacklisted.
+Sonar-scala creates two rule repositories and four built-in quality profiles: `Scalastyle`, `Scapegoat` and a combination of those two `Scalastyle+Scapegoat` and `Recommended by sonar-scala`. The rules in the quality profiles are almost all activated, except for these which are broken in the upstream projects and had to be blacklisted.
```

Comment:
Maybe "two rule*s* repositories" would sound better? _(I changed rule by its plural, rules)_.

Reply:
Also, I will not argue about english with a native speaker...
But I learnt that _these_ means: "a collection of things which are here or near - like I can pick them",
while _those_ means: "a collection of things which are far - like I can point to them".
Given that, for me _those_ sound better than _these_ in this case...
However, I am sure I am making a mistake somewhere - it would be nice if you could explain to me the rationale behind your change.

Reply:
I'm not a native speaker either, but your comments make sense to me - I updated the PR.

Reply:
😮 _I always thought you were british_ - oh well, my fault :+1: